bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/config/OpenSprinklerStationConfig.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.opensprinkler.internal.config;
+
+/**
+ * The {@link OpenSprinklerStationConfig} class defines the configuration options
+ * for the OpenSprinkler Thing.
+ *
+ * @author Chris Graham - Initial contribution
+ */
+public class OpenSprinklerStationConfig {
+    /**
+     * The index of the station the thing is configured to control, starting with 0.
+     */
+    public int stationIndex = -1;
```

Comment:
if `-1` is invalid, why do you set it here?

Reply:
because it's an int. And 0 would not make sense, as it is a valid, but maybe not configured value. It depends on what we want, if the user did not configure this value correctly. I thought, that -1 is the best value to assume in this case? It's like the indexOf method of a JavaScript string, which returns -1, too, if the string was not found (even though the use case here is a little different).